gapic/src/main/com/google/gapid/server/GapisProcess.java

```diff
@@ -96,6 +97,20 @@ protected Exception prepare(ProcessBuilder pb) throws GapiPaths.MissingToolsExce
       args.add(settings.preferences().getAnalyticsClientId());
     }
 
+    // The --experimental-enable-all flag is a sugar flag from the UI. GAPIS knows nothing about it.
+    if (settings.preferences().getEnableAllExperimentalFeatures() || Experimental.enableAll.get()) {
+      // All --experimental-enable-<feature-name> flags must be added here.
+      args.add("--experimental-enable-frame-lifecycle");
+      args.add("--experimental-enable-vulkan-frame-profiler");
+    } else {
+      if (Experimental.enableFrameLifecycle.get()) {
+        args.add("--experimental-enable-frame-lifecycle");
+      }
+      if (Experimental.enableVulkanFrameProfiler.get()) {
+        args.add("--experimental-enable-vulkan-frame-profiler");
+      }
+    }
+
```

Comment:
Can you move this logic into your new `Experimental` class? i.e. have something like `Experimental.getGapisFlags(..)`. This would keep the logic a bit more contained.